src/activities.ts

```diff
@@ -1,3 +1,36 @@
-export async function greet(name: string): Promise<string> {
-  return `Hello, ${name}!`;
+import { Context } from '@temporalio/activity'
+import {
+  FulfillmentResult,
+  fulfillmentService,
+  inventoryService,
```

Comment:
You probably don't want to use global state in your app, better to instantiate these services on worker creation and inject into the activities.